### PR TITLE
fix: inconsistent column in data display

### LINKED ISSUE #4: Inconsistent Column Count in Data Display
The number of data columns in the site is inconsistent.
Screenshot 2024-04-18 at 9 55 59 PM
Screenshot 2024-04-18 at 9 56 12 PM

### PR DESCRIPTION
**Pull request by iamyuu**
closes #4

###### Screenshot

u496COaU 2024-04-19 at 01 49 PM@2x

AJkULvEt 2024-04-19 at 01 49 PM@2x

### CODE REVIEW

File: src/lib/remembergaza_web/controllers/page_html/home.html.heex [hunk 1]
```diff
@@ -17,7 +17,7 @@
     id="victims"
     role="log"
     aria-live="polite"
-    class="flex flex-wrap gap-x-6 justify-center mb-8"
+    class="grid gap-6 grid-cols-[repeat(auto-fill,_minmax(200px,_1fr))] mb-8"
```
Comment (iamyuu) [inline src/lib/remembergaza_web/controllers/page_html/home.html.heex:20]: I'm not sure about `200px` in here, any suggestions?

**Review by msafdev — COMMENTED** [review]
200px should be fine

**Comment by AsadSaleh** [comment]
All good? Anything blocking this PR? @msafdev @ariefrahmansyah

**Review by ariefrahmansyah — APPROVED** [review]
LGTM! Thanks, @iamyuu!

**Pull request merged**